sequencing/vcf_slice/Readme.md

```diff
@@ -1,21 +1,32 @@
-<!-- dx-header -->
 # Slice VCF File(s) (DNAnexus Platform App)
 
-Return a small section of a VCF file (similar to tabix).  For large output, many small regions, or subsetting samples, use subset_vcf instead.
+Return a small section of a VCF file (similar to tabix).  For large output, 
```

Comment:
Make into 1 line

Reply:
Markdown ignores linebreaks, so this change wouldn't do anything.